epictrack-api/migrations/versions/eed10a65e3c3_make_columns_not_nullable.py

```diff
@@ -0,0 +1,90 @@
+"""make columns not nullable
+
+Revision ID: eed10a65e3c3
+Revises: 2257493941e7
+Create Date: 2023-10-23 16:17:27.132349
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = 'eed10a65e3c3'
+down_revision = '2257493941e7'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    with op.batch_alter_table('works', schema=None) as batch_op:
+        batch_op.alter_column('eao_team_id',
+               existing_type=sa.INTEGER(),
```

Comment:
I think this will create migration issues as the existing tables has data in dev and test environment. So it is better to truncate the tables before adding these columns.